src/localization/locales/fi.json

```
@@ -75,18 +75,23 @@
     "errorUploadingExistingTaskAttachment": "Virhe ladattaessa olemassaolevaa tehtävän liitettä",
     "errorListingTaskAttachments": "Virhe haettaessa tehtävän liitteitä",
     "errorDeletingTaskAttachment": "Virhe poistettaessa tehtävän liitettä",
+    "errorListingTaskConnections": "Virhe haettaessa tehtävien riippuvuuksia",
     "errorListingCompanies": "Virhe haettaessa yrityksiä",
     "errorCreatingNewCompany": "Virhe luotaessa uutta yritystä",
     "errorAssigningUserToProject": "Virhe lisättäessä käyttäjää projektiin",
     "errorCreatingProjectMilestone": "Virhe luotaessa uutta projektin virstanpylvästä",
     "errorListingProjectMilestones": "Virhe haettaessa projektin virstanpylväiden luetteloa",
     "errorFindingProjectMilestone": "Virhe haettaessa projektin virstanpylvää",
     "errorListingMilestoneTasks": "Virhe haettaessa virstanpylvään tehtäviä",
+    "errorFindingMilestoneTask": "Virhe haettaessa virstanpylvän tehtävää",
```

Better word for milestone in the context of project management could be 'tavoite'.

```suggestion
    "errorCreatingProjectMilestone": "Virhe luotaessa uutta projektin tavoitetta",
    "errorListingProjectMilestones": "Virhe haettaessa projektin tavoitteita",
    "errorFindingProjectMilestone": "Virhe haettaessa projektin tavoitetta",
    "errorListingMilestoneTasks": "Virhe haettaessa tavoitteen tehtäviä",
    "errorFindingMilestoneTask": "Virhe haettaessa tavoitteen tehtävää",
```